app/controllers/coronavirus_pages_controller.rb

```diff
@@ -35,8 +35,36 @@ def publish
     end
   end
 
+  def reorder
+    coronavirus_page
+    @old_positions = positions
+    if request.post? && params.key?(:section_order_save)
+      @new_positions = JSON.parse(params[:section_order_save])
+      set_positions(@new_positions)
+      if draft_updater.send
+        message = { notice: "Sections were successfully reordered." }
+      else
+        set_positions(@old_positions)
```

Comment:
As per my PR comment, not sure if we would want to roll back? You could just show an error asking the user to click save again.